include/PR/ultratypes.h

```diff
@@ -22,12 +22,6 @@ typedef volatile s64 vs64;
 typedef float  f32;
 typedef double f64;
 
-typedef u32 size_t;
-
-#ifndef NULL
-#define NULL (void*)0
-#endif
-
```

Comment:
libultra defines size_t and NULL on ultratypes.h. Why did you remove them from here?

Reply:
Oh, I was following what OoT has. Didn't think they were actually from `ultratypes`, will add them back.

Reply:
It does some ifdef shenanigans to avoid redefining `size_t`
https://github.com/decompals/ultralib/blob/eddcfcd44e6bf181996c577dc2384e823e999fed/include/PR/ultratypes.h#L60-L69

Tharo did a similar ifdef check on his libc PR, but only for `stddef.h`
https://github.com/zeldaret/oot/pull/1568/files#diff-4e201337732f1fcf8edb60e2ec377650650121fb6078dba6cbc647134b487a4a